rdr_service/alembic/versions/d7321a399c60_adding_new_manifest_fields.py

```diff
@@ -0,0 +1,94 @@
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = 'd7321a399c60'
+down_revision = ('d98dda217034', '0e2d0a24dbea')
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('genomic_aw1_raw', sa.Column('pediatric', sa.String(length=80), nullable=True))
+    op.create_index(op.f('ix_genomic_aw1_raw_pediatric'), 'genomic_aw1_raw', ['pediatric'], unique=False)
+    op.add_column('genomic_aw2_raw', sa.Column('pediatric', sa.String(length=80), nullable=True))
+    op.add_column('genomic_aw2_raw', sa.Column('sequencer', sa.String(length=80), nullable=True))
+    op.create_index(op.f('ix_genomic_aw2_raw_pediatric'), 'genomic_aw2_raw', ['pediatric'], unique=False)
+    op.create_index(op.f('ix_genomic_aw2_raw_sequencer'), 'genomic_aw2_raw', ['sequencer'], unique=False)
+    op.add_column('genomic_aw3_raw', sa.Column('pediatric', sa.String(length=80), nullable=True))
+    op.add_column('genomic_aw3_raw', sa.Column('sequencer', sa.String(length=80), nullable=True))
+    op.create_index(op.f('ix_genomic_aw3_raw_pediatric'), 'genomic_aw3_raw', ['pediatric'], unique=False)
+    op.create_index(op.f('ix_genomic_aw3_raw_sequencer'), 'genomic_aw3_raw', ['sequencer'], unique=False)
+    op.add_column('genomic_aw4_raw', sa.Column('pediatric', sa.String(length=80), nullable=True))
+    op.add_column('genomic_aw4_raw', sa.Column('sequencer', sa.String(length=80), nullable=True))
+    op.create_index(op.f('ix_genomic_aw4_raw_pediatric'), 'genomic_aw4_raw', ['pediatric'], unique=False)
+    op.create_index(op.f('ix_genomic_aw4_raw_sequencer'), 'genomic_aw4_raw', ['sequencer'], unique=False)
+    op.add_column('genomic_aw5_raw', sa.Column('pediatric', sa.String(length=80), nullable=True))
+    op.add_column('genomic_aw5_raw', sa.Column('sequencer', sa.String(length=80), nullable=True))
+    op.create_index(op.f('ix_genomic_aw5_raw_pediatric'), 'genomic_aw5_raw', ['pediatric'], unique=False)
+    op.create_index(op.f('ix_genomic_aw5_raw_sequencer'), 'genomic_aw5_raw', ['sequencer'], unique=False)
+    op.drop_column('genomic_aw5_raw', 'biobank_id_sample_id')
```

Comment:
Did you mean to drop this column? I didn't see the same update to the AW5 model

Reply:
Weird not sure how that hot there, edited the migration.